app/samples/benchmark_littlefs_datalogger/prj.conf

```diff
@@ -0,0 +1,32 @@
+CONFIG_CPP=y
+CONFIG_REQUIRES_FULL_LIBCPP=y
+CONFIG_STD_CPP20=y
+
+CONFIG_F_CORE=y
+CONFIG_F_CORE_OS=y
+
+CONFIG_SERIAL=y
+
+CONFIG_SHELL=y
+CONFIG_FLASH_SHELL=y
+CONFIG_FILE_SYSTEM_SHELL=y
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+
+CONFIG_DEBUG=y
+CONFIG_LOG=y
+CONFIG_LOG_MODE_IMMEDIATE=y
+CONFIG_CBPRINTF_FP_SUPPORT=y
+
+CONFIG_FLASH=y
```

Comment:
you add the snippet in the Cmake but still have these here, is that intentional?

Reply:
Unintentional. Forgot to remove